app/app_test.go

```diff
@@ -327,8 +329,8 @@ func TestDoAdvancedPermissionsMigration(t *testing.T) {
 			model.PERMISSION_MANAGE_SLASH_COMMANDS.Id,
 			model.PERMISSION_MANAGE_OTHERS_SLASH_COMMANDS.Id,
 			model.PERMISSION_MANAGE_WEBHOOKS.Id,
-			model.PERMISSION_MANAGE_PUBLIC_CHANNEL_PROPERTIES.Id,
-			model.PERMISSION_MANAGE_PRIVATE_CHANNEL_PROPERTIES.Id,
+			// model.PERMISSION_MANAGE_PUBLIC_CHANNEL_PROPERTIES.Id,
+			// model.PERMISSION_MANAGE_PRIVATE_CHANNEL_PROPERTIES.Id,
```

Comment:
Why are there changes to this test?

Reply:
The order seems to have changed. I haven't looked into why.

Reply:
Weird. Makes me a bit uncomfortable because I can't see any reason in this branch that it should have changed.

Reply:
resolved by Jesus' changes.